src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -273,8 +273,9 @@ public async Task RefreshAsync(CancellationToken cancellationToken)
                                 // Check if a change has been detected in the key-value registered for refresh
                                 if (change.ChangeType != KeyValueChangeType.None)
                                 {
-                                    logDebugBuilder.AppendLine($"{LoggingConstants.RefreshKeyValueChanged}(key: '{change.Key}', label: '{change.Label}')");
-                                    logInfoBuilder.AppendLine($"{LoggingConstants.RefreshKeyValueSettingUpdated}'{change.Key}' from endpoint {endpoint}");
+                                    string changeTypeString = change.ChangeType == KeyValueChangeType.Modified ? "Modified" : "Deleted";
+                                    logDebugBuilder.AppendLine($"{LoggingConstants.RefreshKeyValueLoaded} Change: {changeTypeString}. Key: '{change.Key}'. Label: '{change.Label}'.");
+                                    logInfoBuilder.AppendLine($"{LoggingConstants.RefreshKeyValueSettingUpdated} Key: '{change.Key}'. Endpoint: [ {endpoint} ].");

```

Comment:
We are trying to establish some structured logging formatting. It's good to keep that logging format in a central place instead of inline here.